### DISCUSSION TITLE
Participate in OpenProducer's community media pilot

### DISCUSSION THREAD

**Discussion by gusaus** [OpenProducer/angelcityjazz#53] [Planning & Strategy]
My agency [OpenProducer](https://www.openproducer.org/) along with partners such as [Open Collective Foundation](https://opencollective.foundation/) (OpenProducer's fiscal sponsor), [NFCB](https://nfcb.org/), [WordPress Training](https://make.wordpress.org/training/) are launching a pilot ([see this overview](https://github.com/OpenProducer/community/wiki/Sustainability#platform-beta)) that will provide training, mentorship and technical assistance to not only manage the website, but implement strategic engagement, content and revenue goals while growing the organizational capacity to sustain over time.

We're still in the process of determining the ongoing [price/service levels](https://github.com/OpenProducer/community/discussions/26) but as discussed we could start by re-allocating the cost savings from not renewing the paid plugins (plus a little bit more). So the following would cover all costs (including ongoing platform updates and maintenance) for the balance of the year:

- make a $500 donation to [the project](https://opencollective.com/openproducer/projects/community-media-pilot/donate) I just set up for the program
- [set up ACA on Open Collective](https://github.com/OpenProducer/angelcityjazz/issues/38) - this is not a dependency, but something we should set up before we actively fundraise (see below)

We're hoping to coordinate a fundraiser using the [same platform](https://fundoss.org/) and [match fund model](https://fundoss.org/democratic-funding) for all participants in our pilot in Q4. Ideally, this will cover all program and platform costs for the following year. 

Again we're still sorting out the details, but as a current client we'll make sure you get the most without spending more than the 1k/yr you've been able to budget for site updates. 

Comment here if there's anything you're still confused about or let me know if there's additional info you need to better evaluate.

_Originally posted by @gusaus in https://github.com/OpenProducer/angelcityjazz/issues/38#issuecomment-1152836938_

**Comment by angelcityjazz** [comment]
Just made the $500 contribution

**Reply by gusaus** [reply]
@angelcityjazz Due to a variety of reasons we've yet to officially launch the program. Hoping to do so over the next few weeks and we can then circle back and pick up on some of the ideas outlined above. 

Just to be clear, the program you already paid into will focus on strategy, building capacity, fundraising, etc, but the platform updates, maintenance, hosting, and additional custom work are not included in that $500 (which I think you already know but wanted to point out again).

We're still trying to determine the monthly cost for updates and maintenance (obviously you're already paying for hosting), but I'm going to grandfather you into the same amount you've been budgeting for if you're able to make the [same financial contribution](https://opencollective.com/openproducer/orders/530171) as you were able to make last year. 

I must say that I'm feeling pretty good about ACJ recouping that cost once we get this campaign up and running.

Followup here or hit me back in an email if you have any questions.
